mails/themes/classic/Readme.md

```diff
@@ -0,0 +1,7 @@
+# Classic theme
+
+This theme was generated based on the PHP templates from the ps_emailgenerator module
+https://github.com/PrestaShop/ps_emailgenerator
+
+To convert them a little script was used, you can clone it from:
+https://github.com/jolelievre/ps_convert_templates
```

Comment:
Maybe move this repository to friends-of-prestashop, or official orga?

Reply:
Well we discussed it with @eternoendless it is a one shot tool used to create the classic theme.
But it is mostly a dev tool for me, and even the classic theme will soon become useless when the new one will be integrated.
So i's not worth it sharing this repo, actually maybe I should even remove it from the Readme it can be confusing.